client/checkout/utils/upe.js

```diff
@@ -376,18 +376,11 @@ export const togglePaymentMethodForCountry = ( upeElement ) => {
 	} else {
 		upeContainer.style.display = 'none';
 		if ( paymentMethodType === selectedPaymentMethod ) {
-			const defaultPaymentMethod = 'card';
 			const cardPaymentForm = document.querySelector(
-				`.wcpay-upe-form[data-payment-method-type="${ defaultPaymentMethod }"]`
+				'input[name="payment_method"][value="woocommerce_payments"]'
```

Comment:
cc @frosso I went back to targeting the Core input element in `togglePaymentMethodForCountry` as well